src/renderer/modals/SystemLanguageAvailable/SystemLanguageAvailableBody.js

```diff
@@ -0,0 +1,139 @@
+// @flow
+import React from "react";
+import { useTranslation } from "react-i18next";
+import { useDispatch } from "react-redux";
+import type { TFunction } from "react-i18next";
+import styled from "styled-components";
+import moment from "moment";
+
+import { ModalBody } from "~/renderer/components/Modal";
+import Box from "~/renderer/components/Box";
+import TrackPage from "~/renderer/analytics/TrackPage";
+import Text from "~/renderer/components/Text";
+import Button from "~/renderer/components/Button";
+import LanguageIcon from "~/renderer/icons/Language";
+
+import { rgba } from "~/renderer/styles/helpers";
+import { languageLabels } from "~/renderer/screens/settings/sections/General/LanguageSelect";
+import { setLanguage } from "~/renderer/actions/settings";
+import { answerLanguageAvailable } from "~/renderer/components/IsSystemLanguageAvailable";
+
+type Props = {
+  data: {
+    currentLanguage: string,
+    osLanguage: string,
+  },
+  onClose: () => void,
+  t: TFunction,
+};
+
+const Title = styled(Text).attrs(() => ({
+  ff: "Inter",
+  fontSize: 18,
+  color: "palette.text.shade100",
+  textAlign: "center",
+  fontWeight: 600,
+  marginBottom: 16,
+}))``;
+
+const Content = styled(Text).attrs(() => ({
+  ff: "Inter",
+  fontSize: 13,
+  color: "palette.text.shade50",
+  textAlign: "center",
+  marginBottom: 13 * 1.4,
```

Comment:
looks like math to me :laughing: 
maybe put the correct value directly

Reply:
done